cmd/fs-v1.go

```diff
@@ -55,6 +51,8 @@ var defaultEtag = "00000000000000000000000000000000-1"
 
 // FSObjects - Implements fs object layer.
 type FSObjects struct {
+	GatewayUnsupported
```

Comment:
could we rename this to be ObjectLayerUnsupported

Reply:
Not right now @poornas